components/courses/app/controllers/admin/courses/test_task_controller.rb

```diff
@@ -21,14 +21,14 @@ def test_task
         @test_task ||= ::Courses::TestTask.find(params[:id])
       end
 
-      def test_tasks
-        @test_tasks ||= current_season.test_tasks.includes(student: :user).preload(mentor: :user)
-      end
-
       def default_redirect
         redirect_to admin_courses_season_test_task_index_path
       end
 
+      def test_tasks
+        @test_tasks ||= ::Courses::TestTask::AllTestTasks.call(current_season)
```

Comment:
I suggest not to duplicate `TestTask` in `AllTestTasks`. 
What about `::Courses::TestTask::All`?